pkg/maps/ctmap/entry.go

```diff
@@ -44,14 +44,19 @@ func (c *CtEntry) GetValuePtr() unsafe.Pointer { return unsafe.Pointer(c) }
 
 // String returns the readable format
 func (c *CtEntry) String() string {
-	return fmt.Sprintf("expires=%d RxPackets=%d RxBytes=%d TxPackets=%d TxBytes=%d Flags=%x RevNAT=%d SourceSecurityID=%d\n",
```

Comment:
Is there a reason the field is called RxBytes previously vs. TxBytes now?

Reply:
For me it seems like a mistake, because...